modules/extensions-spring-support/src/main/java/org/mule/module/extension/internal/config/BaseExtensionBeanDefinitionParser.java

```diff
@@ -38,5 +61,11 @@ public BeanDefinition parse(Element element, ParserContext parserContext)
         return definition;
     }
 
+    /**
+     * Template method to add the component specific parsing logic.
```

Comment:
it's a template method between another template method. Happens all the time, happens on TNT